src/fileReader.cpp

```diff
@@ -52,14 +46,28 @@ void fileReader::readFile(string fileName)
             wordMap[mystring] += 1;
             count++;
         }
+
+        for (auto x : wordMap)
```

Comment:
Can you maybe add some comments to easier explain what this does for in the interview in case he specifically asks one of us about it